cli/internal/terraform/terraform/qemu/modules/instance_group/main.tf

```diff
@@ -53,24 +56,46 @@ resource "libvirt_domain" "instance_group" {
     type        = "pty"
     target_port = "0"
   }
+  lifecycle {
+    ignore_changes = [
+      name, # required. Allow legacy scale sets to keep their old names
+    ]
+  }
```

Comment:
Since QEMU doesn't support upgrades, we can drop the backwards compatibility code here.